inc/patterns/general-404.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * 404 content.
+ */
+return array(
+	'title'      => __( '404 content', 'twentytwentytwo' ),
+	'categories' => array( 'twentytwentytwo-general' ),
+	'content'    => '<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"200","lineHeight":"1"}},"className":"has-text-align-center"} -->
+					<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:200;line-height:1">' . esc_html__( '404', 'twentytwentytwo' ) . '</h2>
```

Comment:
I'm really not sure, but should this have context alongside it?

```suggestion
					<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:200;line-height:1">' . esc_html( _x( '404', 'Error code for a webpage that is not found.', 'twentytwentytwo' ) ) . '</h2>
```

Reply:
I think adding context makes sense. What's the difference between using `_x` and this method: https://github.com/WordPress/twentytwentytwo/blob/trunk/inc/patterns/footer-default.php#L15-L17